### PR TITLE
[2.4] Corrects evolutions configuration keys

### PR DESCRIPTION
**Pull request by studiodev**
Fix in the code and documentation the configuration keys about evolutions (`play.modules.evolutions` -> `play.evolutions`)

Fixes for #4533

### CODE REVIEW

**Comment by typesafehub-validator** [comment]
Hi @julienlafont,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Typesafe Contributors License Agreement:

[http://www.typesafe.com/contribute/cla](http://www.typesafe.com/contribute/cla)

**Comment by studiodev** [comment]
Typesafe Contributor License Agreement signed.

**Comment by julienlafont** [comment]
Ok the PR is linked to this account, sorry :)

File: documentation/manual/detailedTopics/production/cloud/Deploying-CleverCloud.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ The file must contain the following fields:
 
 That field can contain additional configuration like:
 
-`"-Dconfig.resource=clevercloud.conf"`, `"-Dplay.version=2.0.4"` or `"-Dplay.modules.evolutions.autoApply=true"`.
+`"-Dconfig.resource=clevercloud.conf"`, `"-Dplay.version=2.0.4"` or `"-Dplay.evolutions.autoApply=true"`.
```
Comment (dotta) [inline documentation/manual/detailedTopics/production/cloud/Deploying-CleverCloud.md]: Should the play version here be updated to Play 2.4.0?

**Comment by dotta** [comment]
@play-pull-request-validator retrying

File: documentation/manual/detailedTopics/production/cloud/Deploying-CleverCloud.md (hunk 1 above)
Comment (studiodev) [inline documentation/manual/detailedTopics/production/cloud/Deploying-CleverCloud.md]: BTW I'm not sure if this configuration is still required by Clevercloud [(Clevercloud doc)](https://www.clever-cloud.com/doc/java/play-framework-2/#known-problems-with-play-2). Maybe someone should try.

Comment (dotta) [inline documentation/manual/detailedTopics/production/cloud/Deploying-CleverCloud.md]: I see. It would be great to ping someone from clevercloud...

**Comment by marcospereira** [comment]
@studiodev / @julienlafont 

I think one of the accounts did not signed the Typesafe Contributors License Agreement.

Thanks.

**Comment by julienlafont** [comment]
I've resigned the CLA with my 2 accounts.

@play-pull-request-validator retrying plz

**Comment by marcospereira** [comment]
Weird. Maybe @typesafehub-validator and @play-pull-request-validator  must be both quoted.

**Comment by jroper** [comment]
Not sure why the CLA validator isn't working in this case, I'll look into it.

**Comment by julienlafont** [comment]
I close this PR. I will retry with my other account.

**Comment by marcospereira** [comment]
@jroper should we backport this?

**Comment by jroper** [comment]
Backported to 2.4.x: 70e99a7

**Pull request merged**